include/nlohmann/detail/workarounds/borland.hpp

```diff
@@ -0,0 +1,66 @@
+#pragma once
+
+#if defined(__clang__) && defined(__BORLANDC__) && defined(_WIN32) && !defined(_WIN64)
+  #if (__clang_major__ * 10000 + __clang_minor__ * 100 + __clang_patchlevel__) >= 30301 && (__clang_major__ * 10000 + __clang_minor__ * 100 + __clang_patchlevel__) < 30400
+
+      #pragma message ("Fixing shortcomings in Borland C++ Builder Clang (bcc32c) compiler [3.3.1, 3.4.0)")
+      #include <cfloat>
+      #include <cmath>
+      #define _NLOHMANN_JSON_SKIP_COMPILER_CHECK_ 1
+      #define _NLOHMANN_JSON_FIX_CONSTEXPR_FOR_CHAR_TRAITS_EOF_ -1
+
+        namespace std {
```

Comment:
Adding definitions to `std` being forbidden by the standard, I guess the chances of this PR being merged are quite low...

But you know that already :p.

Reply:
Sure did. :-) I removed the workaround file in the new commit.